tests/tests_integration/tests_diff_version/tests_common/test_commands.py

```diff
@@ -0,0 +1,29 @@
+from th2_data_services import Data
+
+def test_break_iteration_get_events(demo_events_from_data_source: Data):
+    iterated_one_item = False
```

Comment:
Let's also add some test description.
At least that the test related to the bug + jira link